examples/hono-openapi.md

```diff
@@ -46,12 +36,11 @@ Use `describeRoute` for route documentation and validation:
 
 ```ts
 import { Hono } from 'hono'
-import { describeRoute } from 'hono-openapi'
-// You can import these for your preferred validation library
 import {
+  describeRoute,
   resolver,
   validator as vValidator,
```

Comment:
Just curious. Should we rename `validator` to `vValidator`. I think `validator` is okay because it does not depend on a specific validator.